docs/source/sample_datasets.rst

```diff
@@ -30,22 +30,48 @@ Creating random data
 
 Random data can be generated in the following way. It should be kept in mind
 that the data generated in this way can only be used for simple testing as it
-has no physical significance.
+has no physical significance. You can then load the data through the
+:ref:`api documentation`.
+
+**Raw file:**
 
 .. code-block:: python
-      
+
+    # Create sample raw file
     import numpy as np
     real_data = np.random.randn(16, 16, 16, 16).astype("float32")
     real_data.tofile("/tmp/real_raw_file.raw")
 
-Now you can load the data through the :ref:`api documentation` in the following way:
+.. code-block:: python
+
+    # Load the file
+    from libertem.api import Context
+    if __name__ == '__main__':
+      ctx = Context()
+      ds = ctx.load("raw", path="/tmp/real_raw_file.raw", scan_size=(16, 16), dtype="float32", detector_size=(16, 16))
+
+**HDF5 file:**
 
 .. code-block:: python
-    
+
+    # Create sample HDF5 file
+    import h5py
+    import numpy as np
+    file = h5py.File('hdf5_sample.h5','w')
+    dataset = file.create_dataset("dataset",(16,16,16,16), h5py.h5t.STD_I32BE)
+    dataset = file['/dataset']
+    data = np.random.randn(16,16,16,16).astype("float32")
+    dataset[...] = data
+    file.close()
+
+.. code-block:: python
+
+    # Load the file
     from libertem.api import Context
-    ctx = Context()
-    ds = ctx.load("raw", path="/tmp/real_raw_file.raw", scan_size=(16, 16), dtype="float32", detector_size=(16, 16))
-    
+    if __name__ == '__main__':
+      ctx = Context()
+      ds = ctx.load("hdf5", path="./hdf5_sample.h5", ds_path="/dataset", tileshape=(1, 4, 16, 16))
```

Comment:
The `tileshape` parameter can be removed, as it is ignored in the current master branch.